drivers/ieee802154/ieee802154_cc13xx_cc26xx.c

```diff
@@ -0,0 +1,735 @@
+{
+	struct ieee802154_cc13xx_cc26xx_data *drv_data = get_dev_data(dev);
+
+	for (int i = 0; i < CC13XX_CC26XX_NUM_RX_BUF; i++) {
+		if (drv_data->rx_entry[i].status == DATA_ENTRY_FINISHED) {
+			u8_t len = drv_data->rx_data[i][0];
+			u16_t fcf = sys_get_le16(&drv_data->rx_data[i][1]);
+			u8_t seq = drv_data->rx_data[i][3];
+			u8_t corr = drv_data->rx_data[i][len--] & 0x3F;
+			u8_t rssi = drv_data->rx_data[i][len--];
+
+			LOG_DBG("len: %u, seq: %u, fcf: %04x", len, seq, fcf);
+			LOG_DBG("rssi: %u lqi: %u", rssi, corr);
+
+			if ((fcf & 0x7) == 0x2) {
+				LOG_DBG("ack: seq=%u", seq);
+				drv_data->rx_entry[i].status =
+					DATA_ENTRY_PENDING;
+				if (drv_data->cmd_ieee_rx_ack.seqNo == seq) {
+					k_sem_give(&drv_data->ack);
+				}
+				continue;
+			}
+
+			struct net_pkt *pkt = net_pkt_rx_alloc_with_buffer(
+				drv_data->iface, len, AF_UNSPEC, 0, K_NO_WAIT);
+			if (!pkt) {
+				LOG_WRN("Cannot allocate packet");
+				continue;
+			}
+
+			if (net_pkt_write(pkt, drv_data->rx_data[i] + 1, len)) {
+				LOG_WRN("Cannot write packet");
+				net_pkt_unref(pkt);
+				continue;
+			}
+
+			drv_data->rx_entry[i].status = DATA_ENTRY_PENDING;
+
+			net_pkt_set_ieee802154_lqi(pkt, corr);
+			net_pkt_set_ieee802154_rssi(pkt, rssi);
+
+			if (net_recv_data(drv_data->iface, pkt)) {
+				LOG_WRN("Packet dropped");
+				net_pkt_unref(pkt);
+			}
+
+		} else if (drv_data->rx_entry[i].status ==
+			   DATA_ENTRY_UNFINISHED) {
+			LOG_WRN("Frame not finished");
+			drv_data->rx_entry[i].status = DATA_ENTRY_PENDING;
+		}
+	}
+}
+
+static void ieee802154_cc13xx_cc26xx_rx(void *arg1, void *arg2, void *arg3)
+{
+	struct ieee802154_cc13xx_cc26xx_data *drv_data = get_dev_data(arg1);
+
+	ARG_UNUSED(arg2);
+	ARG_UNUSED(arg3);
+
+	while (true) {
+		k_sem_take(&drv_data->rx_done, K_FOREVER);
+		ieee802154_cc13xx_cc26xx_rx_done(arg1);
+	}
+}
+
+static int ieee802154_cc13xx_cc26xx_start(struct device *dev)
+{
+	ARG_UNUSED(dev);
+	return 0;
+}
+
+static int ieee802154_cc13xx_cc26xx_stop(struct device *dev)
+{
+	ARG_UNUSED(dev);
+	RFCDoorbellSendTo(CMDR_DIR_CMD(CMD_STOP));
+	return 0;
+}
+
+static void ieee802154_cc13xx_cc26xx_cpe0_isr(void *arg)
+{
+	struct ieee802154_cc13xx_cc26xx_data *drv_data = get_dev_data(arg);
+
+	u32_t flags = RFCCpeIntGetAndClear(IRQ_RX_ENTRY_DONE | IRQ_TX_DONE);
+
+	if (flags & IRQ_RX_ENTRY_DONE) {
+		k_sem_give(&drv_data->rx_done);
+	}
+
+	if (flags & IRQ_TX_DONE) {
+		k_sem_give(&drv_data->tx_done);
+	}
+}
+
+static void ieee802154_cc13xx_cc26xx_cpe1_isr(void *arg)
+{
+	ARG_UNUSED(arg);
+
+	u32_t flags =
+		RFCCpeIntGetAndClear(IRQ_RX_BUF_FULL | IRQ_INTERNAL_ERROR);
+
+	if (flags & IRQ_RX_BUF_FULL) {
+		LOG_WRN("Receive buffer full");
+	}
+
+	if (flags & IRQ_INTERNAL_ERROR) {
+		LOG_ERR("Internal error");
+	}
+}
+
+static void ieee802154_cc13xx_cc26xx_data_init(struct device *dev)
+{
+	struct ieee802154_cc13xx_cc26xx_data *drv_data = get_dev_data(dev);
+
+	u8_t *mac = (u8_t *)(FCFG1_BASE + FCFG1_O_MAC_15_4_0);
```

Comment:
move that line one line up, and have an empty line before the if.

The common rule is like:
u8_t foo;

foo = stuff;
if (foo) { ...

or: 

u8_t foo = stuff;

if (foo) {